gateway/webhook/webhook.go

```diff
@@ -244,6 +262,19 @@ func (webhook *HandleT) RequestHandler(w http.ResponseWriter, r *http.Request) {
 	ss.Report(webhook.stats)
 }
 
+func getWebhookFailureReason(err string, statusCode int) string {
+	switch {
+	case err == response.RequestBodyTooLarge:
+		return response.RequestBodyTooLarge
+	case statusCode != 0:
+		return response.SourceTransformerNonSuccessResponse
+	case err != "":
+		return response.SourceTransformerResponseError
```

Comment:
[optional] Not needed at all of course, but I never got the chance to comment about `fallthrough` so... here we are :joy:

```go
case err != "":
    fallthrough
default:
    return response.SourceTransformerResponseError
```